src/app/content/highlights/components/ShowMyHighlightsStyles.tsx

```diff
@@ -69,8 +70,11 @@ export const ShowMyHighlightsBody = styled(PopupBody)`
 
 // tslint:disable-next-line:variable-name
 export const HighlightWrapper = styled.div`
-  margin: 1.6rem ${popupPadding}rem;
+  margin: ${mobilePaddingSides}rem ${popupPadding}rem;
```

Comment:
just because it happens to be the same value doesn't mean it makes sense to use a variable called `mobilePaddingSides` for _desktop vertical margin_